python/valis/routes/target.py

```diff
@@ -195,6 +196,26 @@ async def get_catalogs(self, sdss_id: int = Path(title="The sdss_id of the targe
 
         return response_data
 
+    @router.get('/parents/{parent_catalog}/{sdss_id}',
+                dependencies=[Depends(get_pw_db)],
```

Comment:
Can we shorten `parent_catalog` to just `catalog` here?

Reply:
Done